src/PKSim.Core/Snapshots/Mappers/PopulationAnalysisFieldMapper.cs

```diff
@@ -4,18 +4,19 @@
 using OSPSuite.Core.Domain.UnitSystem;
 using OSPSuite.Utility.Extensions;
 using PKSim.Core.Model.PopulationAnalyses;
+using PKSim.Core.Repositories;
 
 namespace PKSim.Core.Snapshots.Mappers
 {
    public class PopulationAnalysisFieldMapper : ObjectBaseSnapshotMapperBase<IPopulationAnalysisField, PopulationAnalysisField>
    {
       private readonly GroupingDefinitionMapper _groupingDefinitionMapper;
-      private readonly IDimensionFactory _dimensionFactory;
+      private readonly IDimensionRepository _dimensionRepository;
 
-      public PopulationAnalysisFieldMapper(GroupingDefinitionMapper groupingDefinitionMapper, IDimensionFactory dimensionFactory)
+      public PopulationAnalysisFieldMapper(GroupingDefinitionMapper groupingDefinitionMapper, IDimensionRepository dimensionRepository)
       {
          _groupingDefinitionMapper = groupingDefinitionMapper;
-         _dimensionFactory = dimensionFactory;
+         _dimensionRepository = dimensionRepository;
```

Comment:
This is the big one. Most of the conversion is always with dimensions used in columns. Howvever here, it was a dimension that is only use for PK Parameter in pop analysis